### PR TITLE
Add release notes for 2.13.0

### PR DESCRIPTION
**Pull request by bunchesofdonald**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,121 @@
...
+Six months ago, we deprecated references to `orion` in our codebase. In this release, we're removing those references. If you still have references to `ORION` in your profile, run `prefect config validate` to automatically convert all of the settings in your *current* profile to the new names!
+
+For example:
+```bash
+❯ prefect config validate
+Updated 'PREFECT_ORION_DATABASE_CONNECTION_URL' to 'PREFECT_API_DATABASE_CONNECTION_URL'.
+Configuration valid!
+```
+
+#### Below is a full guide to the changes:
```
Comment (serinamarie) [inline RELEASE-NOTES.md:48]: @billpalombi Are we okay with the "full guide" here?

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,121 @@
...
+
+### Introducing `Global concurrency limits`
+
+Control task execution and system stability with Prefect's new Global Concurrency Limits and Rate Limits. 
+
+- **Concurrency Limits:** Manage task execution efficiently, controlling how many tasks can run simultaneously. Ideal for optimizing resource usage and customizing task execution.
+
+- **Rate Limits:** Ensure system stability by governing the frequency of requests or operations. Perfect for preventing overuse, ensuring fairness, and handling errors gracefully.
+
+Choose Concurrency Limits for resource optimization and task management, and opt for Rate Limits to maintain system stability and fair access to services. To begin using global concurrency limits check out our [guide](https://docs.prefect.io/guides/global-concurrency-limits/).
```
Comment (serinamarie) [inline RELEASE-NOTES.md:13]: the Prefect concepts should be lowercased

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,121 @@
...
+- Add Kubernetes guide for deploying worker to Azure AKS — https://github.com/PrefectHQ/prefect/pull/10575
+- Fix broken link to docker guide with redirect and harmonize naming — https://github.com/PrefectHQ/prefect/pull/10624
+- Add information on `--no-prompt` and `PREFECT_CLI_PROMPT` to deployment documentation — https://github.com/PrefectHQ/prefect/pull/10600
+- Remove link to nowhere in api-keys.md — https://github.com/PrefectHQ/prefect/pull/10658
+- Update quickstart screenshots and add cli log output — https://github.com/PrefectHQ/prefect/pull/10659
+- Add documentation for global concurrency limits — https://github.com/PrefectHQ/prefect/pull/10496
+- Update deployment concept documentation to emphasize server-side deployment — https://github.com/PrefectHQ/prefect/pull/10615
+- Add work pool and worker status documentation — https://github.com/PrefectHQ/prefect/pull/10654
+
+**All changes**: https://github.com/PrefectHQ/prefect/compare/2.12.1...preview
```
Comment (serinamarie) [inline RELEASE-NOTES.md:117]:
```suggestion
**All changes**: https://github.com/PrefectHQ/prefect/compare/2.12.1...2.13.0
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,121 @@
 # Prefect Release Notes
 
+## Release 2.13.0
+
+### Introducing `Global concurrency limits`
+
+Control task execution and system stability with Prefect's new Global Concurrency Limits and Rate Limits.
```
Comment (billpalombi) [inline RELEASE-NOTES.md:7]:
```suggestion
Control task execution and system stability with Prefect's new Global Concurrency and Rate Limits. 
```

File: RELEASE-NOTES.md (hunk 1 above)
Comment (billpalombi) [reply RELEASE-NOTES.md:48]: I don't know where else we would put it, so I think here is fine.

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,121 @@
 # Prefect Release Notes
 
+## Release 2.13.0
+
+### Introducing `Global concurrency limits`
```
Comment (billpalombi) [inline RELEASE-NOTES.md:5]:
```suggestion
### Introducing global concurrency limits
```

**Review by billpalombi — COMMENTED** [review]
A couple proposed edits

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,121 @@
...
+
+### Documentation
+- Add Kubernetes guide for deploying worker to Azure AKS — https://github.com/PrefectHQ/prefect/pull/10575
+- Fix broken link to docker guide with redirect and harmonize naming — https://github.com/PrefectHQ/prefect/pull/10624
+- Add information on `--no-prompt` and `PREFECT_CLI_PROMPT` to deployment documentation — https://github.com/PrefectHQ/prefect/pull/10600
+- Remove link to nowhere in api-keys.md — https://github.com/PrefectHQ/prefect/pull/10658
+- Update quickstart screenshots and add cli log output — https://github.com/PrefectHQ/prefect/pull/10659
+- Add documentation for global concurrency limits — https://github.com/PrefectHQ/prefect/pull/10496
+- Update deployment concept documentation to emphasize server-side deployment — https://github.com/PrefectHQ/prefect/pull/10615
+- Add work pool and worker status documentation — https://github.com/PrefectHQ/prefect/pull/10654
```
Comment (serinamarie) [inline RELEASE-NOTES.md:115]: I think this can be moved to the highlight

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,121 @@
...
+### Fixes
+- Fix an issue with `prefect server start` in Windows - https://github.com/PrefectHQ/prefect/pull/10547
+
+### Documentation
+- Add Kubernetes guide for deploying worker to Azure AKS — https://github.com/PrefectHQ/prefect/pull/10575
+- Fix broken link to docker guide with redirect and harmonize naming — https://github.com/PrefectHQ/prefect/pull/10624
+- Add information on `--no-prompt` and `PREFECT_CLI_PROMPT` to deployment documentation — https://github.com/PrefectHQ/prefect/pull/10600
+- Remove link to nowhere in api-keys.md — https://github.com/PrefectHQ/prefect/pull/10658
+- Update quickstart screenshots and add cli log output — https://github.com/PrefectHQ/prefect/pull/10659
+- Add documentation for global concurrency limits — https://github.com/PrefectHQ/prefect/pull/10496
```
Comment (serinamarie) [inline RELEASE-NOTES.md:113]: this can move to highlight

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,121 @@
...
+
+Control task execution and system stability with Prefect's new Global Concurrency Limits and Rate Limits. 
+
+- **Concurrency Limits:** Manage task execution efficiently, controlling how many tasks can run simultaneously. Ideal for optimizing resource usage and customizing task execution.
+
+- **Rate Limits:** Ensure system stability by governing the frequency of requests or operations. Perfect for preventing overuse, ensuring fairness, and handling errors gracefully.
+
+Choose Concurrency Limits for resource optimization and task management, and opt for Rate Limits to maintain system stability and fair access to services. To begin using global concurrency limits check out our [guide](https://docs.prefect.io/guides/global-concurrency-limits/).
+
+### Introducing work pool and worker status
```
Comment (serinamarie) [inline RELEASE-NOTES.md:15]: we can reference the documentation for this

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,121 @@
...
+    - Class `OrionRouter` → `PrefectRouter`
+    - Class `OrionAPIRoute` → `PrefectAPIRoute`
+    - Class `OrionDBInterface` → `PrefectDBInterface`
+    - Class `OrionClient` → `PrefectClient`
+
+See the following pull request for details:
+- Remove deprecated `orion` references — https://github.com/PrefectHQ/prefect/pull/10642
+
+### Fixes
+- Fix an issue with `prefect server start` in Windows - https://github.com/PrefectHQ/prefect/pull/10547
```
Comment (serinamarie) [inline RELEASE-NOTES.md:105]:
```suggestion
- Fix an issue with `prefect server start` on Windows - https://github.com/PrefectHQ/prefect/pull/10547
```

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -1,5 +1,121 @@
...
+
+### Documentation
+- Add Kubernetes guide for deploying worker to Azure AKS — https://github.com/PrefectHQ/prefect/pull/10575
+- Fix broken link to docker guide with redirect and harmonize naming — https://github.com/PrefectHQ/prefect/pull/10624
+- Add information on `--no-prompt` and `PREFECT_CLI_PROMPT` to deployment documentation — https://github.com/PrefectHQ/prefect/pull/10600
+- Remove link to nowhere in api-keys.md — https://github.com/PrefectHQ/prefect/pull/10658
```
Comment (serinamarie) [inline RELEASE-NOTES.md:111]:
```suggestion
- Remove invalid link in API keys documentation — https://github.com/PrefectHQ/prefect/pull/10658
```

File: RELEASE-NOTES.md [hunk 11]
```diff
@@ -1,5 +1,121 @@
...
+- Update quickstart screenshots and add cli log output — https://github.com/PrefectHQ/prefect/pull/10659
```
Comment (serinamarie) [inline RELEASE-NOTES.md:112]:
```suggestion
- Update screenshots and CLI log output in quickstart documentation — https://github.com/PrefectHQ/prefect/pull/10659
```

File: RELEASE-NOTES.md [hunk 12]
```diff
@@ -1,5 +1,121 @@
...
+- Fix an issue with `prefect server start` on Windows - https://github.com/PrefectHQ/prefect/pull/10547
+
+### Documentation
+- Add Kubernetes guide for deploying worker to Azure AKS — https://github.com/PrefectHQ/prefect/pull/10575
+- Fix broken link to docker guide with redirect and harmonize naming — https://github.com/PrefectHQ/prefect/pull/10624
+- Add information on `--no-prompt` and `PREFECT_CLI_PROMPT` to deployment documentation — https://github.com/PrefectHQ/prefect/pull/10600
+- Remove invalid link in API keys documentation — https://github.com/PrefectHQ/prefect/pull/10658
+- Update quickstart screenshots and add cli log output — https://github.com/PrefectHQ/prefect/pull/10659
+- Add documentation for global concurrency limits — https://github.com/PrefectHQ/prefect/pull/10496
+- Update deployment concept documentation to emphasize server-side deployment — https://github.com/PrefectHQ/prefect/pull/10615
```
Comment (serinamarie) [inline RELEASE-NOTES.md:114]: I think this could be moved up

File: RELEASE-NOTES.md [hunk 13]
```diff
@@ -1,5 +1,123 @@
...
+
+See the following pull request for details:
+- https://github.com/PrefectHQ/prefect/pull/10496
+
+### Introducing work pool and worker status
+
+Work pools and workers are critical components of Prefect's distributed execution model. To help you monitor and manage your work pools and workers, we've added status indicators to the Prefect UI.
+
+Work pools can now have one of three statuses:
+- `Ready` -  at least on online worker is polling the work pool and the work pool is ready to accept work.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:23]:
```suggestion
- `Ready` -  at least one online worker is polling the work pool and the work pool is ready to accept work.
```

**Review by billpalombi — APPROVED** [review]
lgtm

**Review by serinamarie — APPROVED** [review]
LGTM 🚀

**Pull request merged**